### ISSUE TITLE
Provide Allowlist / Exclusion List

### ISSUE THREAD

**Issue by xRuffKez** [hagezi/dns-blocklists#2751]
Hey @hagezi 

Is it possible to provide the Exclusion List from your build as Allowlist for people who uses various blocklists?
This might help, to make the services i.e. AdguardHome more bulletproof upon FPs.

**Comment by hagezi** [comment]
In short: No

My build system uses its own syntax for excluding domains. This makes it possible to do things like remove `*. example.com`, but not `a.example.com` or remove `a[4-digit number].example.[com|org...]`. Then there are rules that are excluded for certain list versions.

Furthermore, it makes a difference whether you use an allowlist to remove domains for a build or as an allowlist for an AdBlocker.
Example: `example.com` removes `example.com` and `www.example.com` from my lists, but if you convert this to Adblock `@@||example.com^`, `*.example.com` is allowed.
I had made my normal allowlist from the build available at the time, people then used it as an allowlist in AdBlockers and complained that - for the reason mentioned above - too much was being unblocked. I then removed it again.
